python/ray/serve/tests/test_failure.py

```diff
@@ -209,6 +210,41 @@ def __call__(self):
                 time.sleep(0.1)
 
 
+def test_create_backend_idempotent(serve_instance):
+    serve.init()
+
+    def f():
+        pass
+
+    controller = serve.api._get_controller()
+
+    replica_config = ReplicaConfig(f)
+    backend_config = BackendConfig({"num_replicas": 1})
+
+    for i in range(10):
+        controller.create_backend.remote("my_backend", backend_config,
+                                         replica_config)
```

Comment:
I think these calls are actually failing because you didn't specify `f`. You won't see the errors unless you wrap these in `ray.get()`, which you should probably do here.

Reply:
Ah, thanks for the reminder.  I just wrapped it in `ray.get()`, and it didn't produce any errors.  I don't know if this is what you were referring to, but `f` is included in `replica_config`, does it need to be specified somewhere else too?

Reply:
Oh, right, I was reading this as if it were the public API.